proofs/sectorbuilder/interface.go

```diff
@@ -47,11 +47,11 @@ type SectorBuilder interface {
 	// which will fit into a newly-provisioned staged sector.
 	GetMaxUserBytesPerStagedSector() (uint64, error)
 
-	// GeneratePoST creates a proof-of-spacetime for the replicas managed by
+	// GeneratePoSt creates a proof-of-spacetime for the replicas managed by
 	// the SectorBuilder. Its output includes the proof-of-spacetime proof which
 	// is posted to the blockchain along with any faults. The proof can be
 	// verified by the VerifyPoSt method on the Verifier interface.
-	GeneratePoST(GeneratePoSTRequest) (GeneratePoSTResponse, error)
+	GeneratePoSt(GeneratePoStRequest) (GeneratePoStResponse, error)
```

Comment:
new casing is less ELiTE

Reply:
Had to do what I had to do to [conform to the spec](https://github.com/filecoin-project/specs/blob/master/drafts/proof-of-spacetime.md#proof-of-spacetime).